rust/agents/relayer/src/relayer.rs

```diff
@@ -68,6 +68,44 @@ impl Agent for Relayer {
             blacklist,
         })
     }
+
+    #[allow(clippy::async_yields_async)]
+    async fn run(&self) -> Instrumented<JoinHandle<Result<()>>> {
```

Comment:
Pay extra attention here because there was a pretty significant conflict and I think I got it right but worth the eyes.